modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -66,3 +66,26 @@ Defaults to 4 seconds.
 The presence ping gather job parameter changes the timeout setting for gathering the presence ping, in seconds.
 Adjust the [systemitem]``java.salt_presence_ping_gather_job_timeout`` parameter.
 Defaults to 1 second.
+
+
+== Disabling the Salt Mine
+
+In older versions, {productname} used a tool called Salt mine to check minion availability.
+With the introduction of more efficient tools in {productname}{nbsp}3.2, it is no longer required.
```

Comment:
Maybe here we could talk a bit about the new mechanism, something like: " now the Server has a Taskomatic task that will ping only those minions that appear offline since at least 12 hours, with the target of pinging all of them at least once every 24 hours (by default)." (that's actually how it works, so that's why the events load is much smaller)

Reply:
We can also say that the newly registered minions does not have the Salt mine enabled by default anymore (since 3.2.6), but for the already registered is recommended to disabled it, especially if the minion count is big.